app/models/sub_service_request.rb

```diff
@@ -62,6 +62,10 @@ def consult_arranged_date=(date)
     write_attribute(:consult_arranged_date, date.present? ? Time.strptime(date, "%m/%d/%Y") : nil)
   end
 
+  def display_service_requester_name
```

Comment:
If service requester is an identity, then we can do something equivalent using `ssr.service_requester.full_name`